flux_local/image.py

```diff
@@ -56,7 +72,8 @@ def add_image(name: str, doc: dict[str, Any]) -> None:
 
             Updates the image set with the images found in the document.
             """
-            images = _extract_images(doc)
+            kind = doc.get("kind")
```

Comment:
The typing above says str but this can be none. If it's always present then use ["kind"]

Reply:
I updated and added the str type to it, should work